kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/ColumnMapping.java

```diff
@@ -93,6 +100,68 @@ public static StructType convertToPhysicalSchema(
         }
     }
 
+    /** Returns the physical name for a given {@link StructField} */
+    public static String getPhysicalName(StructField field) {
+        if (field.getMetadata().contains(COLUMN_MAPPING_PHYSICAL_NAME_KEY)) {
+            return (String) field
+                    .getMetadata()
+                    .get(COLUMN_MAPPING_PHYSICAL_NAME_KEY);
+        } else {
+            return field.getName();
+        }
+    }
+
+    public static void verifyColumnMappingChange(
+            Map<String, String> oldConfig,
+            Map<String, String> newConfig) {
+        verifyColumnMappingChange(oldConfig, newConfig, true /* isNewTable */);
+    }
```

Comment:
nit: looks like this method can be removed?

Reply:
done